pkg/plugins/config/config.go

```diff
@@ -1,16 +1,13 @@
 package config
 
 import (
-	"github.com/grafana/grafana-azure-sdk-go/azsettings"
-
-	"github.com/grafana/grafana/pkg/plugins/log"
 	"github.com/grafana/grafana/pkg/services/featuremgmt"
 	"github.com/grafana/grafana/pkg/setting"
 )
 
-type Cfg struct {
-	log log.Logger
-
+// PluginsCfg is the configuration for the plugin management system.
+// It includes settings which are used to configure different components of plugin management.
+type PluginsCfg struct {
```

Comment:
I'd go all the way and call this `PluginManagementCfg` to make it clear and not having to go to the comment. WDYT?